src/plugins/intel_cpu/src/config.cpp

```diff
@@ -89,9 +89,19 @@ void Config::readProperties(const ov::AnyMap& prop, const ModelType modelType) {
                                    ". Expected only ov::Affinity::CORE/NUMA/HYBRID_AWARE.");
                 }
             }
+            if (key == ov::num_streams.name()) {
+                ov::Any value = val.as<std::string>();
+                auto streams_value = value.as<ov::streams::Num>();
+                if (streams_value == ov::streams::NUMA) {
+                    latencyThreadingMode = Config::LatencyThreadingMode::PER_NUMA_NODE;
+                } else if (streams_value == ov::streams::AUTO) {
+                    hintPerfMode = ov::hint::PerformanceMode::THROUGHPUT;
```

Comment:
Let keep both `ov::streams::AUTO` and `ov::hint::performance_mode` during core.set_property() and process them during compile_model().

Reply:
Because `readProperties` is called in core.set_property(), so in either case, `num_streams` will be set first, and then `PerformanceMode` will be set.